_posts/research/2020-04-09-cooperative_multirobot_systems.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Cooperative Multi-Robot Systems
+description: Reinforcement Learning for Cooperative Control
+author: Joewie J. Koh
+permalink: research/cooperative_multirobot_systems.html
+image:
+    feature: research/multirobot/cooperative_manipulation.png
+    size: 70%
+excerpt_separator: <!-- More -->
+---
+
+Multi-robot systems offer vastly increased competencies for robotic applications. Multiple robotic agents can, through cooperative control, accomplish common goals beyond the capabilities of single-robot systems. In particular, a cooperative multi-robot system has capabilities that are more than the sum of its parts, e.g., robots can cooperate to manipulate or transport objects that are too large or heavy for a single robot to manage.
+
+<!-- More -->
+
+{% include image.html url="research/multirobot/cooperative_manipulation.png" max-width="70%" description="Two robotic arms cooperatively manipulating the same object." %}
+
+Robot control has traditionally been accomplished with model-based methods, which require domain knowledge of system kinematics and dynamics and are therefore difficult to employ in unstructured environment. Being less reliant on system knowledge, model-free deep reinforcement learning (RL) has seen recent interest and success, but most applications have been in single-agent settings. For multi-robot systems, these methods are inadequate: the increased number of degrees of freedom and coupling of multiple kinematic and dynamic constraints pose challenges for both model-based control and RL. One approach taken to mitigate these issues in RL involves deviating from the centralized learning paradigm and training multiple agents in parallel, but ignoring agent interactions leads to instability in the policy learning process.
+
+{% include image.html url="research/multirobot/cooperative_transport.jpg" max-width="40%" description="Two mobile robots cooperatively transporting an object." %}
+
+We thus approach the prolem of cooperative multi-robot control with distributed multi-agent RL: while avoiding centralized learning (and its limitations), we mitigate the adverse impact of agent interactions on the policy learning process by allowing for information sharing between agents.
```

Comment:
You thus what? It is not clear from the paragraph above what you want to do.

Reply:
I have read this whole paragraph and I still don't have any idea about what you work on.